src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -166,6 +166,15 @@ pip3 install -r requirements.txt
 $ cwltool --version
 /usr/local/bin/cwltool ${this.cwltoolVersion}
 \`\`\`
+
+#### Part 5 - Install Nextflow (Optional)
+Nextflow workflows cannot run on the Dockstore CLI. Users must run them directly by using the Nextflow command line tool.
```

Comment:
I would say "The Dockstore CLI does not run Nextflow workflows", but I don't feel super strongly about it. I would argue that the CLI is not a platform that you run things on.